.github/workflows/canary.yml

```diff
@@ -2,23 +2,23 @@ name: electron-nightly Canary
 
 on:
   schedule:
-    - cron: '15 8 * * *'
+    - cron: "15 8 * * *"
 
 jobs:
   build:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [windows-latest, macOS-latest, ubuntu-latest]
+        os: [macOS-latest, ubuntu-latest, windows-latest]
```

Comment:
question: this reordering seems unnecessary?

Reply:
oh yeah i think it is. i removed windows and added it back and i think that messed up the ordering.

Reply:
[aa10ea9](https://github.com/electron/electron-packager/pull/1539/commits/aa10ea9d6d0dd803b879c746389929fbb18610d5)